packages/DropdownMenu/src/components/Item/Item.js

```diff
@@ -28,27 +31,39 @@ const propTypes = {
 const defaultProps = {
   isDestructive: false,
   onClick: () => {},
+  onKeyDown: () => {},
   onClose: () => {},
   onShowConfirmation: () => {},
   renderConfirmation: null,
 };
 
 const Item = props => {
-  const { children, isDestructive, onClick, onClose, onShowConfirmation, renderConfirmation, ...moreProps } = props;
+  const {
+    children,
+    isDestructive,
+    onClick,
+    onClose,
+    onShowConfirmation,
+    renderConfirmation,
+    onKeyDown,
+    ...moreProps
+  } = props;
 
   const handleClickItem = () => {
-    onClick();
+    onClick(children);
     onClose();
   };
 
   const itemProps = {
     onClick: renderConfirmation !== null ? onShowConfirmation : handleClickItem,
     role: "menuitem",
+    className: "dropdown-item",
```

Comment:
Adding the classname so that the on onFocus event can get the right element - https://github.com/acl-services/paprika/pull/318/files#diff-edd13737a415dcf236bf45816da474cbR42